compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -1933,28 +1933,29 @@ TR::Register *OMR::X86::TreeEvaluator::integerDualMulEvaluator(TR::Node *node, T
 
 TR::Register *OMR::X86::TreeEvaluator::integerMulEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   TR::Register *targetRegister = 0;
-   TR::Node     *firstChild     = node->getFirstChild();
-   TR::Node     *secondChild    = node->getSecondChild();
-   bool          nodeIs64Bit    = TR::TreeEvaluator::getNodeIs64Bit(node, cg);
-   intptrj_t    value;
-
+   TR::Register *targetRegister  = 0;
+   TR::Node     *firstChild      = node->getFirstChild();
+   TR::Node     *secondChild     = node->getSecondChild();
+   TR::DataType dataType         = secondChild->getDataType();
+   bool         nodeIs64Bit      = TR::TreeEvaluator::getNodeIs64Bit(node, cg);
+   bool firstChildRefCountIsZero = false;
+   intptr_t     value;
+   
    if (node->isDualCyclic())
       {
       return TR::TreeEvaluator::integerDualMulEvaluator(node, cg);
       }
-
+   
```

Comment:
Please delete the white spaces at this line.

Reply:
Done